### ISSUE TITLE
Improve violation reporter

### ISSUE THREAD

**Issue by scurker** [nickcolley/jest-axe#63]
Violations currently may not have enough context, depending on the rule that's being violated.

*Example:*

```
<div id="dialog" aria-labelledby="foo" aria-describedby="desc">
  <h1 id="bar">Title</h1>
  <p id="desc">Description</p>
</div>
```

*Returns:*

```
Expected the HTML found at $('#dialog') to have no violations:

<div id="dialog" aria-labelledby="foo" aria-describedby="desc">
  <h1 id="bar">Title</h1>
  <p id="desc">Description</p>
</div>

Received:

"ARIA attributes must conform to valid values (aria-valid-attr-value)"

Try fixing it with this help: https://dequeuniversity.com/rules/axe/3.1/aria-valid-attr-value?application=axeAPI
```

If I'm not well versed on accessibility, I may not know exactly what to look for. Which attribute? What's invalid? However, we're in luck! For each `node` that has a violation, axe has a failure summary that is more specific on what to fix.

> Fix all of the following:
> Invalid ARIA attribute value: aria-labelledby="foo"

Including this failure summary as part of the violations reporter would provide more clear direction on what to investigate.

**Comment by dakebl** [comment]
Hello @scurker, I've raised PR #64 to add the failure summary to the output for each node and updated some of the formatting to make it a little less noisy when it does.
Let me know what you think?
@nickcolley?

**Comment by nickcolley** [comment]
This will be released in the next release tracked by this milestone: https://github.com/nickcolley/jest-axe/milestone/1

Thanks for raising this @scurker and @dakebl for fixing.
